src/views/lecturer/EditCreateCourseForm.vue

```diff
@@ -177,7 +152,33 @@
             course.value.startDate = "2020-06-01";
             course.value.endDate = "2020-08-31";
 
-            const errorBag: ErrorBag = reactive(new ErrorBag());
+            var errorBag: ErrorBag = reactive(new ErrorBag());
```

Comment:
Why lower the restriction from `const` to `var` on the errorBag?

Reply:
Vue can not watch on changes within lists inside of objects, even if the objects are reactive.
So to trigger an update (without this.$forceUpdate which we can't even really do due to type restrictions), I need to create a new ErrorBag object, instead of using "replaceAll"